testData/parser/MapTypes.erl

```diff
@@ -1,9 +1,9 @@
 -type map1() :: #{ binary() => integer() }.
 -type map2() :: #{ one | two => [number()] }.
 
--spec func(Opt, M) -> #{'status' => S, 'c' => integer()} when
+-spec func(Opt, M) -> #{'status' := S, 'c' => integer()} when
```

Comment:
Both the old and new syntax are mixed in the test
On Fri, 3 Jun 2016 at 20:17, Sergey Ignatov <notifications@github.com>
wrote:

> In testData/parser/MapTypes.erl
> <https://github.com/ignatov/intellij-erlang/pull/741#discussion_r65752799>
> :
>
> > @@ -1,9 +1,9 @@
> >  -type map1() :: #{ binary() => integer() }.
> >  -type map2() :: #{ one | two => [number()] }.
> >
> > --spec func(Opt, M) -> #{'status' => S, 'c' => integer()} when
> > +-spec func(Opt, M) -> #{'status' := S, 'c' => integer()} when
>
> I suppose it's better to keep a test for old syntax as well
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ignatov/intellij-erlang/pull/741/files/8fefc62c751e8a3ceb6bf6dc88c1ec2ac1e2320e..dd82d83e9fba7cd317a2968a0f5c64d259f309e0#r65752799>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAWx77BkL1NUCTjKkUAVOZ2C2HHlKHMqks5qIG-pgaJpZM4ItVmc>
> .
>